README.md

```diff
@@ -148,6 +148,62 @@ If in the above example, the `doSomething` causes an error, subsequently, `rende
 
 If in the above example, none of the functions causes an error, the `handleError` will never be invoked.
 
+### Extension Points
+
+In addition to the (optional) wrapper function which can be invoked prior to the `once` function, pipeline creators can expose named extension points. These extension points allow users of a pipeline to inject additional functions that will be called right before or right after an extension point. To keep the extension points independent from the implementation (i.e. the name of the function), pipeline authors should use the `expose(name)` function to expose a particular extension point.
+
+Example:
+
+```js
+new pipeline()
+  .before(doSomething).expose('init')
+  .once(render)
+  .after(cleanup).expose('cleanup')
+  .after(done);
+```
+
+In this example, two extension points, `init` and `cleanup` have been defined. Note how the name of the extension point can be the same as the name of the function (i.e. `cleanup`), but does not have to be the same (i.e. `init` vs. `doSomething`).
+
+#### Common Extension Points
+
+The creation of extension points are the responsibility of the pipeline author, but in order to standardize extension points, following common names have been established:
```

Comment:
Grammar: 
- The creation (...) **is** the responsibility of (...)
- (...) **the** following common names (...)

Reply:
```suggestion
The creation of extension points is the responsibility of the pipeline author, but in order to standardize extension points, the following common names have been established:
```